src/hooks/useDebounce.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useState } from 'react';
 
 // https://usehooks.com/useDebounce/
-export function useDebounce(value: any, delay: number) {
+export function useDebounce(value: string, delay: number) {
```

Comment:
This should probably be a generic type
```suggestion
export function useDebounce<T>(value: T, delay: number) {
```